desktop/test/render-visdiff.js

```diff
@@ -61,15 +64,16 @@ app.on('ready', () => {
     })
   })
 
-  ipcMain.on('visdiff-ready', ev => renderNext(ev.sender))
-
   const scriptPath = resolveRoot('dist', 'visdiff.bundle.js')
   for (let i = 0; i < WORKER_COUNT; i++) {
     setTimeout(() => {
       const workerWin = new BrowserWindow({show: false, width: CANVAS_SIZE, height: CANVAS_SIZE})
+      console.log('Created new worker window', i)
       // TODO: once we're on electron v1.2.3, try ready-to-show event.
```

Comment:
dunno if we want to at this moment but we are using a newer electron than 1.2.3 so maybe we could change this?